SPECS/python3/python3.spec

```diff
@@ -28,6 +28,7 @@ Patch4:         CVE-2024-6232.patch
 Patch5:         CVE-2024-8088.patch
 Patch6:         CVE-2024-4032.patch
 Patch7:         CVE-2024-11168.patch
+Patch8:         CVE-2024-6923.patch
```

Comment:
This spec does not automatically apply all patches. So can you please add a line to apply this patch with `%patch8 -p1` below in the %prep section